libs/src/services/creatorNode/index.js

```diff
@@ -17,6 +17,31 @@ class CreatorNode {
    */
   static getSecondaries (endpoints) { return endpoints ? endpoints.split(',').slice(1) : [] }
 
+  /**
+   * Checks if a download is available from provided creator node endpoints
+   * @param {string} endpoints creator node endpoints
+   * @param {number} trackId
+   */
+  static async checkIfDownloadAvailable (endpoints, trackId) {
+    for (const endpoint in endpoints.split(',')) {
```

Comment:
I'd rather not because the string is what's provided at the protocol-level. I think it makes more sense for the client to not have to understand the format of the value if i can just pass it back and forth and have the protocol-layer stuff handle it.